### PR TITLE
Add MAVLinkServer

### PR DESCRIPTION
**Pull request by patrickelectric**
Working fine with SITL, the problem when connecting with an autopilot is probably a missing stream-request in the router. That should be configured with command line as mavp2p.
https://github.com/bluerobotics/mavlink-server/issues/30

### CODE REVIEW

**Comment by Williangalvani** [comment]
@patrickelectric can you rebase as master was broken?

**Comment by patrickelectric** [comment]
> @patrickelectric can you rebase as master was broken?

Rebased

File: core/tools/mavlink_server/bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# Immediately exit on errors
+set -e
+
+VERSION="master"
```
Comment (joaoantoniocardoso) [inline core/tools/mavlink_server/bootstrap.sh:6]: just a reminder to fixate a version once we validate it

**Review by Williangalvani — COMMENTED** [review]
looks good and it is working as expected here, but I agree with @joaoantoniocardoso that we should change the version from master

**Pull request merged**